src/pages/People.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
```

Comment:
I think this is a more correct syntax... or not...😅
```suggestion
  const [isError, setIsError] = useState('');
```

Reply:
No, we use name starting with `is` only if it's a `boolean` value
In this case we can rename it to `errorMessage`